showcase/src/app/demo-ui/pages/table/demo-table-page.component.html

```diff
@@ -33,6 +33,14 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 		<showcase-table-with-fixed-header></showcase-table-with-fixed-header>
 	</example-viewer>
 
+	<example-viewer
+		exampleTitle="SHOWCASE.DEMO.TABLE.WITH_FIXED_ACTIONS"
+		filesPath="table/with-sticky-actions/table"
```

Comment:
This path should also be changed to "fixed"... because the app now throws an exception:

`GET http://localhost:3000/assets/examples/table/with-sticky-actions/table.html 404 (Not Found)`